src/pages/Lend/LendByCategoryPage.vue

```diff
@@ -7,6 +7,13 @@
 		<!-- MFI Recommendations Section -->
 		<div v-if="mfiRecommendationsExp" class="tw-max-w-5xl tw-mx-auto lg:tw-px-6">
 			<m-f-i-hero />
+
+			<mfi-loans-wrapper
```

Comment:
Does this need to be behind an experiment?

Reply:
This (`mfiRecommendationsExp`) is the exp if I'm not mistaken

Reply:
Ah yup, I missed the parent.